frontend2/src/content/ManageContent.ts

```diff
@@ -0,0 +1,53 @@
+import type {
+  DebuggingTipsPageKey,
+  TourneyPageKey,
+  QuickStartedPageKey,
+  ResourcesPageKey,
+  CommonIssuesPageKey,
+} from "./ContentStruct";
+
+import {
+  BC24_QUICKSTART,
+  BC24_RESOURCES,
+  BC24_TOURNAMENTS,
+  BC24_DEBUGGINGTIPS,
+  BC24_COMMONISSUES,
+} from "./bc24";
+import { BC23_TOURNAMENTS } from "./bc23";
+import { BC22_TOURNAMENTS } from "./bc22";
+
+export const defaultQuickStartText = BC24_QUICKSTART;
```

Comment:
I saw some issues with certain text fields showing "2024" even when we're on the 2022/23 version of this page (and, in general, mix-ups with years). I think defaulting content pages is, in general, not preferable. We should discuss this at this week's hack session and see if we can all come to an agreement on how we want to manage/handle duplicate content.